Fixes for small issues found at the hackathon (#433)
openstack-k8s-operators/data-plane-adoption: docs_dev/assemblies/development_environment.adoc

> === Create a workload to adopt

Reviewer: ```suggestion
=== Creating a workload to adopt
```

> To be able to run `openstack` commands from the host without actually
> installing the package and copying the configuration file from the VM
> we will create an alias:

Reviewer: ```suggestion
To run `openstack` commands from the host without 
installing the package and copying the configuration file from the virtual machine, create an alias:
```

openstack secret store --name testSecret --payload 'TestPayload'
```

== Performing the adoption procedure

> To make our life easier when performing the adoption procedure, we can

Reviewer: I suggest something like: "To simplify the adoption procedure, copy the deployment passwords that you use in the..."
